Source/PINDiskCache.m

```diff
@@ -201,6 +204,33 @@ - (instancetype)initWithName:(NSString *)name
                      ageLimit:60 * 60 * 24 * 30]; // 30 days by default
```

Comment:
Oh I see these were never constants! Now that we are using them in two places, let's create constants that we can use in both places.